### PR TITLE
Update for redis configuration

### PR DESCRIPTION
**Pull request by vzabaznov**
Add documentation about new options that we release in 2.4.0

## Purpose of this pull request

This pull request (PR) updates redis configuration with a new option that will be part of a new 2.4.0 release.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/config-guide/redis/redis-pg-cache.html

whatsnew
Added new configuration options to [Use Redis for Magento page and default cache](https://devdocs.magento.com/guides/v2.4/config-guide/redis/redis-pg-cache.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.4/config-guide/redis/redis-pg-cache.md [hunk 1]
```diff
@@ -1 +1,188 @@
...
+
+|Command line parameter|Parameter|Meaning|Default value|
+|--- |--- |--- |--- |
+|cache-backend-redis-server|server|Fully qualified hostname, IP address, or an absolute path to a UNIX socket. The default value of 127.0.0.1 indicates Redis is installed on the Magento server.|127.0.0.1|
+|cache-backend-redis-port|port|Redis server listen port|6379|
+|cache-backend-redis-db|database|Required if you use Redis for both the default and full-page cache. You must specify the database number of one of the caches; the other cache uses 0 by default.<br/><br/>Important: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
+|cache-backend-redis-password|password|Configuring a Redis password enables one of its built-in security features: the auth command, which requires clients to authenticate to access the database. The password is configured directly in Redis's configuration file, /etc/redis/redis.conf, which you should still have open from the previous step.||
```
Comment (rogyar) [inline src/guides/v2.4/config-guide/redis/redis-pg-cache.md:33]: > which you should still have open from the previous step

This part might be a little bit confusing for a reader. Once I got to this part I was looking for a previous step where we had a deal with `/etc/redis/redis.conf` but didn't found anything specific. So we may clarify this statement. 

Thank you

**Comment by dobooth** [comment]
Hi @vzabaznov , Check my comment in #7330 about properly breaking symlinks.

**Comment by vzabaznov** [comment]
@dobooth done

@rogyar changes you want me to made is not related to a new part of documentation, thats just old text from 2.3

File: src/guides/v2.4/config-guide/redis/redis-pg-cache.md (hunk 1 above)
Comment (dobooth) [reply src/guides/v2.4/config-guide/redis/redis-pg-cache.md]: I removed that phrase, as it wasn't needed.

**Comment by hostep** [comment]
@vzabaznov: can we have soms sort of explanation what all these `preload_keys` are exactly? How we can find these? And which keys needs to be added here? What is recommended? Stuff like that?

(for some reason Github doesn't allow me to add a single line review anymore, so I'm giving my feedback in a comment here)

**Comment by vzabaznov** [comment]
@hostep nice catch, will do

**Comment by dobooth** [comment]
@vzabaznov Is this ready to go?

**Comment by vzabaznov** [comment]
@dobooth I've made needed changes, think its ready to go

**Comment by dobooth** [comment]
running tests

**Pull request merged**